README.md

```diff
@@ -48,53 +50,41 @@ The extension is available on the Chrome Web Store, Firefox Add-ons, and Microso
-cd github-file-explorer-icons
-pnpm install
-```
+<details>
+<summary>Chrome</summary>
 
-#### Chrome
+1. Unzip the `catppuccin-github-icons-extension-<version>-chrome.zip` asset from the latest release.
+2. Open the Extensions page by navigating to `chrome://extensions`.
+3. Enable Developer Mode (if not already enabled) by flipping the toggle switch labeled **Developer mode**.
+4. Click the **Load unpacked** button and select the unzipped directory from Step 1.
 
-1. Run `pnpm build`.
-1. Open the Extensions page by navigating to `chrome://extensions`.
-1. Enable Developer Mode by flipping the toggle switch labeled **Developer mode**.
-1. Click the **Load unpacked** button and select the `dist/chrome-mv3` directory from the cloned repository.
+</details>
 
-#### Firefox
+<details>
+<summary>Firefox</summary>
 
-1. Run `pnpm zip:firefox`.
 1. Open the Add-ons page by navigating to `about:addons`.
-1. Click the cog/settings icon in line with the "Manage Your Extensions" heading, and select **Debug Add-ons**.
-1. Click the **Load Temporary Add-on...** button and select the `dist/catppuccin-github-icons-extension-<version>-firefox.zip` file from the cloned repository.
+2. Click the cog/settings icon in line with the "Manage Your Extensions" heading, and select **Debug Add-ons**.
+3. Click the **Load Temporary Add-on...** button and select the `catppuccin-github-icons-extension-<version>-firefox.zip` asset from the latest release.
 
-## Development
-
-### Setup
-
-```bash
-git clone --recurse-submodules https://github.com/catppuccin/github-file-explorer-icons.git
-cd github-file-explorer-icons
-pnpm install
-```
+</details>
 
-### Usage
+## Customization
 
-To launch a browser instance with the extension loaded and a GitHub page already opened, run `pnpm dev` for Chrome or `pnpm dev:firefox` for Firefox. Please note that it takes a while to open since it re-copies all icons from the submodule to the public folder.
+This extension aims to provide the same configuration options as [catppuccin/vscode-icons](https://github.com/catppuccin/vscode-icons) does. The flavor of icons, custom associations, as well as the `monochrome` & `specificFolders` options, can be customized in the extension's popup window (click the extension icon in your browser toolbar to reveal the popup).
 
-To build the extension, run either:
+## 🙋 FAQ
 
-- `pnpm build`, where the compiled extension will be available at `dist/chrome-mv3`...
-- or `pnpm zip:firefox`, where the compiled and _zipped_ extension can be found at `dist/catppuccin-github-file-explorer-icons-<version>-firefox.zip`.
+- Q: **_"How can I enable this extension on a custom domain?"_**\
+  A: See [fregante.github.io/webext-permission-toggle](https://fregante.github.io/webext-permission-toggle/?name=Catppuccin+for+GitHub+File+Explorer+Icons&icon=https://raw.githubusercontent.com/catppuccin/github-file-explorer-icons/main/src/public/icon-128.png).
```

Comment:
You might want to use a [`short_name`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/short_name) in your manifest so that users get a better UI in the browser, without affecting the Chrome Web Store.


<table>
<tr>
	<th>Before
	<th>After
<tr>
	<td><img width="343" alt="Screenshot 1" src="https://github.com/user-attachments/assets/e1b913e8-ad30-45e0-b51c-c9bf464f339e">
	<td><img width="343" alt="Screenshot 2" src="https://github.com/user-attachments/assets/8815c686-b87e-426b-8125-d83a1a20b46a">
</table>

And also in `webext-permission-toggle`'s option object you can further improve the UI by specifying `"Enable Catppuccin on this domain"` without "for GitHub"

Reply:
That's fair, but we also have a Catppuccin extension for theming the browser itself ([Chrome](https://github.com/catppuccin/chrome), [Firefox](https://github.com/catppuccin/firefox)). I fear it would be confusing.

Reply:
Made an issue for renaming. For the record, the extension I was inspired by to make this one is still called "Material Icons *for GitHub*"!